src/app/admin/admin-products/admin-products.component.scss

```diff
@@ -1,3 +1,20 @@
 table {
   width: 100%;
 }
+.mat-header-cell {
+  font-family: Roboto, 'Helvetica Neue', sans-serif;
+  font-size: 1rem;
+  color: #000000;
```

Comment:
keep all colors in a scss variable file so that when you want to change colors you dont have to do a search across all of your code. also instead of 000000 use white